frappe/comm/doctype/sms_settings/sms_settings.json

```diff
@@ -62,7 +62,7 @@
  "links": [],
  "modified": "2021-09-21 19:45:26.809793",
  "modified_by": "Administrator",
- "module": "Core",
+ "module": "Comm",
```

Comment:
```suggestion
 "module": "Communications",
```

Reply:
Addressed in [force-push](https://github.com/frappe/frappe/compare/ac242bc6c0c766467dd79629af24797c0bb46cc5..989d70f975d6faa0e6ddbe6618cb2d116173dd85)